<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:info/1 2024-10-04 09:57:27.697 ***</b>
=CRASH REPORT==== 4-Oct-2024::09:57:27.697397 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: &lt;0.244563.0&gt;
    registered_name: my_test_name
    exception exit: {&lt;0.244561.0&gt;,stopped_info}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [&lt;0.244561.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3312
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-04 09:57:29.821 ***</b>
=ERROR REPORT==== 4-Oct-2024::09:57:29.821450 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244595.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-04 09:57:29.822 ***</b>
=CRASH REPORT==== 4-Oct-2024::09:57:29.821898 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244598.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244595.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244595.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-04 09:58:09.718 ***</b>
=ERROR REPORT==== 4-Oct-2024::09:58:09.718817 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248945.0&gt;,
                               {#Ref&lt;0.2543334558.769392641.100087&gt;,1006474,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.2543334558.769392641.100087&gt;,1006474,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-04 09:58:09.719 ***</b>
=CRASH REPORT==== 4-Oct-2024::09:58:09.719172 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248966.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2543334558.769392641.100087&gt;,1006474,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248945.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4508
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_global_registered_parent/1 2024-10-04 09:58:09.766 ***</b>
=CRASH REPORT==== 4-Oct-2024::09:58:09.766154 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global/2
    pid: &lt;0.248977.0&gt;
    registered_name: []
    exception exit: {&lt;0.248975.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [&lt;0.248975.0&gt;]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248975.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3428
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-10-04 10:02:45.270 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:02:45.270620 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254770.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2543334558.770965505.19338&gt;,5371,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254768.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28118
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-04 10:02:54.296 ***</b>
=ERROR REPORT==== 4-Oct-2024::10:02:54.295455 ===
** Generic server <0.256049.0> terminating 
** Last message in was {nodedown,'peer_down_crash-413220-11810@5c666b68e18d'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-413220-11810",
                              peer_down => crash},
                            'peer_down_crash-413220-11810@5c666b68e18d',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-413220-11810",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDVjNjY2YjY4ZTE4ZAAD6DEAAAAAZv+doA=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-413220-11810@5c666b68e18d'}},
                            {<0.256033.0>,#Ref<0.2543334558.763101187.147562>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-413220-11810@5c666b68e18d'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-10-04 10:02:54.296 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:02:54.296405 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256049.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-413220-11810@5c666b68e18d'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.256033.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 11231
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-04 10:02:54.891 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:02:53.889628 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-515074-11810@5c666b68e18d' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-04 10:02:55.783 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:02:55.783366 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-515010-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-04 10:02:55.811 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:02:55.811267 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-515106-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-04 10:03:11.315 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:10.310366 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-413700-11810@5c666b68e18d' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-04 10:03:11.717 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:11.717387 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-435555-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-04 10:03:11.738 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:11.738329 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-390789-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:13.909 ***</b>
=ERROR REPORT==== 4-Oct-2024::10:03:13.909608 ===
** Generic server <0.256416.0> terminating 
** Last message in was {'EXIT',<0.256406.0>,
                               {#Ref<0.2543334558.770965505.36739>,1495439,ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-413828-11810",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-413828-11810@5c666b68e18d',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-413828-11810",
                                     "-user","peer"],
                                    #Port<0.3123>,undefined,<<>>,running,
                                    {<0.256406.0>,
                                     #Ref<0.2543334558.763101187.149819>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2543334558.770965505.36739>,1495439,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:13.910 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:03:13.910234 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256416.0>
    registered_name: []
    exception exit: {#Ref<0.2543334558.770965505.36739>,1495439,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256406.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3123>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 14149
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:14.519 ***</b>
=ERROR REPORT==== 4-Oct-2024::10:03:14.518926 ===
** Generic server <0.256441.0> terminating 
** Last message in was {'EXIT',<0.256401.0>,
                           {#Ref<0.2543334558.763625474.77145>,2108425,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "VBSMRYBUEUIYJGXDKPQY","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@5c666b68e18d.2024-10-04_09.47.43/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-391141-11810",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@5c666b68e18d.2024-10-04_09.47.43/make_test_dir.stdlib_test.logs/run.2024-10-04_09.48.18/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-391141-11810",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-391141-11810@5c666b68e18d',
                                    "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-391141-11810",
                                     "-setcookie","VBSMRYBUEUIYJGXDKPQY",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@5c666b68e18d.2024-10-04_09.47.43/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-391141-11810",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@5c666b68e18d.2024-10-04_09.47.43/make_test_dir.stdlib_test.logs/run.2024-10-04_09.48.18/log_private/",
                                     "-user","peer"],
                                    #Port<0.3145>,undefined,<<>>,running,
                                    {<0.256401.0>,
                                     #Ref<0.2543334558.770965505.37071>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2543334558.763625474.77145>,2108425,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:14.520 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:03:14.519724 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256441.0>
    registered_name: []
    exception exit: {#Ref<0.2543334558.763625474.77145>,2108425,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256401.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3145>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20909
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:14.925 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:13.923728 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-413796-11810@5c666b68e18d' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:15.097 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:15.097453 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-435779-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:15.103 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:15.103246 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-413892-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-04 10:03:15.307 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:03:15.307348 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-413860-11810@5c666b68e18d'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-04 10:14:06.395 ***</b>
=ERROR REPORT==== 4-Oct-2024::10:14:06.395522 ===
** Generic server <0.303199.0> terminating 
** Last message in was {'EXIT',<0.303196.0>,
                               {#Ref<0.2543334558.806617089.78847>,350727,
                                [{child1,<0.303205.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.303199.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.303205.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459170],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2543334558.806617089.78847>,350727,
    [{child1,<0.303205.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-04 10:14:06.396 ***</b>
=ERROR REPORT==== 4-Oct-2024::10:14:06.395529 ===
** Generic server <0.303198.0> terminating 
** Last message in was {'EXIT',<0.303196.0>,
                               {#Ref<0.2543334558.806617089.78847>,350727,
                                [{child1,<0.303205.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.303198.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.303204.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459170],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2543334558.806617089.78847>,350727,
    [{child1,<0.303205.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-04 10:14:06.396 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:14:06.396143 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.303199.0>
    registered_name: []
    exception exit: {#Ref<0.2543334558.806617089.78847>,350727,
                     [{child1,<0.303205.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.303196.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14900
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-04 10:14:06.396 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:14:06.396531 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.303198.0>
    registered_name: []
    exception exit: {#Ref<0.2543334558.806617089.78847>,350727,
                     [{child1,<0.303205.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.303196.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 14988
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-10-04 10:14:49.840 ***</b>
=CRASH REPORT==== 4-Oct-2024::10:14:49.840539 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.403356.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.403351.0>]
    message_queue_len: 0
    messages: []
    links: [<0.403357.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3994
  neighbours:
    neighbour:
      pid: <0.403357.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.403356.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2024-10-04 10:40:18.695 ***</b>
=WARNING REPORT==== 4-Oct-2024::10:40:18.694502 ===
Process <0.406810.0> on node 'test_server@5c666b68e18d' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Oct 04 2024 09:48:18<!--/date--><br />
</div>
</center>
</body>
</html>
